pkgs/development/node-packages/aliases.nix

```diff
@@ -86,7 +86,7 @@ mapAliases {
   inherit (pkgs) configurable-http-proxy; # added 2023-08-19
   copy-webpack-plugin = throw "copy-webpack-plugin was removed because it is a JS library, so your project should lock it with a JS package manager instead."; # Added 2024-12-16
   inherit (pkgs) cordova; # added 2023-08-18
-  inherit (pkgs) create-react-app; # added 2023-09-25
+  create-react-app = throw "create-react-app was removed because it was deprecated. Upstream suggests using a framework for React."; # added 2023-09-25
```

Comment:
Should I update the "added" comment?

Reply:
node-packages/aliases.nix is for making old `nodePackages.foo` usages point to the top-level `foo` package.

So if you only add the `throw` to the node aliases package, it will only throw if you use `nodePackages.foo`. `foo` will not throw you custom message.

You should move the `throw` part to `pkgs/by-name/aliases.nix`